fusor-ember-cli/app/templates/configure-environment.hbs

```diff
@@ -95,12 +95,14 @@
       {{/each}}
     {{/if}}
 
-    <div style='margin-top:15px;'>
+    <div style='margin:15px 0px;'>
       <button class="btn btn-default" {{action 'newEnvironment'}} disabled={{isStarted}}>
         New Environment Path
       </button>
     </div>
 
+    {{error-message errorMsg=errorMsg}}
+
```

Comment:
This shows the error message at the bottom of the screen as well. The BZ suggested to show a message to "scroll up to see error message", but I suggest just to display it at the bottom as well.